engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/impl/ExternalTaskRestServiceImpl.java

```diff
@@ -151,6 +147,10 @@ public BatchDto setRetriesAsync(SetRetriesForExternalTasksDto retriesDto) {
 
   }
 
+  @Override public List<String> getTopicNames(boolean withLockedTasks, boolean withUnlockedTasks, boolean withRetriesLeft) {
+    return processEngine.getExternalTaskService().getTopicNames(withLockedTasks,withUnlockedTasks,withRetriesLeft);
```

Comment:
Lines of code must not exceed 100 characters. If lines with method signatures exceed 100 characters, each parameter must be on a new line. See [Coding Style Java General Formatting](https://github.com/camunda/camunda-bpm-dev-docs/blob/master/development/Coding-Style-Java.md#general-formatting).

Reply:
Could you please add spaces in the parameter list?

Reply:
Please place the annotation above the method declaration – not on the same line. See [Coding Style Java Keywords](https://github.com/camunda/camunda-bpm-dev-docs/blob/master/development/Coding-Style-Java.md#java-keywords).